docs/user-guide/source-hydrator.md

```diff
@@ -279,8 +279,7 @@ hydrator will not be able to use a project-scoped push secret and will require a
 
 ### Credential Templates
 
-Credential templates allow a single credential to be used for multiple repositories. The source hydrator does not 
-currently support credential templates. You will need a separate credential for each repository.
+Credential templates allow a single credential to be used for multiple repositories. The source hydrator supports credential templates. For more information please refer [credential-template](private-repositories.md#credential-templates)
```

Comment:
I'd move this section out from under `## Limitations` since it's now a feature instead of a limitation.

I think it makes sense to link out to the credential-template page, but I'd also recommend adding an example Secret here for easy reference.